docs/content/preview/yugabyte-platform/anywhere-automation/anywhere-cli.md

```diff
@@ -0,0 +1,273 @@
+---
+title: yba CLI
+headerTitle: YugabyteDB Anywhere CLI
+linkTitle: yba CLI
+description: Use YugabyteDB Anywhere CLI to create and manage universes.
+headcontent: Install yba, configure default settings, and set up autocompletion
+earlyAccess: /preview/releases/versioning/#feature-availability
+menu:
+  preview_yugabyte-platform:
+    parent: anywhere-automation
+    identifier: anywhere-cli
+    weight: 50
+type: docs
+rightNav:
+  hideH4: true
+---
+
+The [YugabyteDB Anywhere (YBA) Command Line Interface](https://github.com/yugabyte/yugabyte-db/tree/master/managed/yba-cli) (yba) is an open source tool that enables you to interact with YBA accounts using commands in your command-line shell. With minimal configuration, you can start running commands from the command line that implement functionality equivalent to that provided by the browser-based YBA interface.
+
+## Install yba
+
+To install yba, download the appropriate binary from the yba-cli GitHub project [releases](https://github.com/yugabyte/yba-cli/releases).
+
+On MacOS and Linux, you can also install yba using [Homebrew](https://brew.sh) by entering the following command:
+
+```sh
+brew install yugabyte/tap/yba
+```
+
+For help, enter the following command:
+
+```sh
+yba help
+```
+
+### Upgrade yba
+
+To upgrade yba using Homebrew, enter the following commands:
+
+```sh
+brew update && brew upgrade yba
+```
+
+### Configure yba
+
+Using yba CLI requires providing, at minimum, an [API token](../#authentication).
+
+You can pass the token as a flag when running yba commands. For example:
+
+```sh
+yba universe list --apiToken "eyJ..."
+```
+
+For convenience, you can configure yba with a default API token as follows:
+
+- Use the `auth` command to write the token to a YAML configuration file, as follows:
+
+  ```sh
+  yba auth
+  ```
+
+  At the prompt, paste your API token and press Enter.
+
+  By default, this writes the value to the file `.yba-cli.yaml` under your `$HOME` directory.
```

Comment:
By default, this writes the value to the file ->
By default, this writes the value to the CLI configuration file `$HOME/.yba-cli.yaml`.